CHANGELOG.md

```diff
@@ -25,13 +25,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
   logged in [#1508](https://github.com/greenbone/gsa/pull/1508)
 
 ### Fixed
+- Fix showing ScanConfig trends [#1582](https://github.com/greenbone/gsa/pull/1582) [#1554](https://github.com/greenbone/gsa/pull/1554)
 - Show HostsCount at SectionTitle [#1576](https://github.com/greenbone/gsa/pull/1576)
 - Fixed creating target from hosts [#1575](https://github.com/greenbone/gsa/pull/1575)
 - Fixed CVSSBaseCalculatorv2 [#1572](https://github.com/greenbone/gsa/pull/1572)
 - Fix title in Operating Systems table of Reports [#1567](https://github.com/greenbone/gsa/pull/1567)
 - Fix displaying only usable report formats at report download [#1565](https://github.com/greenbone/gsa/pull/1565)
 - Fix handling schedule_periods ("once" option) in TaskDialog [#1563](https://github.com/greenbone/gsa/pull/1563)
-- Fix showing inactive VerifyIcon at ReportFormats detailspage [#1554](https://github.com/greenbone/gsa/pull/1554)
```

Comment:
Why did you remove the link? Was this by intention?

Reply:
No idea, how this happened. Maybe a leftover from the merge conflict I had to deal with...